### DISCUSSION TITLE
No release events running in fork

### DISCUSSION THREAD

**Discussion by vadi2** [community/community#25174] [Actions]
<p>I’m working</p>
<p>I’ve already tried <code>on: release</code> as well as filters like <code>creasted</code> and <code>published</code>, but the action still fails to run when I create a new release in my fork. Is this event supported for forks? The documentation doesn’t mention any limitations in this regard.</p>
<p><a href="https://github.com/vadi2/ClinicArrivals/blob/patch-1/.github/workflows/publish-release.yml" rel="nofollow noopener">https://github.com/vadi2/ClinicArrivals/blob/patch-1/.github/workflows/publish-release.yml</a> is my workflow and <a href="https://github.com/vadi2/ClinicArrivals/actions?query=workflow%3A.github%2Fworkflows%2Fpublish-release.yml" rel="nofollow noopener">https://github.com/vadi2/ClinicArrivals/actions?query=workflow%3A.github%2Fworkflows%2Fpublish-release.yml</a> shows it stopped running as soon as I switched it to <code>on: release</code>.</p>
<p>How can I get the release event to fire in my fork? I’d like to ensure the workflow is correct before PRing it to the main repository.</p>

**Comment by vadi2** [comment]
<p>No release events are running in the branch of a main repo either, and I’m creating releases manually.</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/vadi2">@vadi2</a> ,</p>
<p>Sure, release event is also supported on forked repositories.</p>
<p>I checked the workflows in your forked repository, I think the problem you are facing is same as that in <a href="https://github.community/t/new-github-action-not-displayed-on-the-actions-tab/17932/2">this ticket</a>.</p>
<p>As I mentioned in that ticket, you need to pay attention to the following points:</p>
<p><strong>1)</strong> After you have created a new workflow, this workflow will not be listed on " <strong>Actions</strong>" tab until it is triggered for the first time.<br>
<strong>2)</strong> The <strong>GITHUB_REF</strong> of the release event is a tag ref, and Github will only reference the workflow YAML files on this tag ref, and the tag ref is created from the specified branch (master in your cases).<br>
<strong>3)</strong> You can’t use the event occurs on a branch/tag ref to trigger the workflows located on another branch/tag ref.<br>
<strong>4)</strong> Do not confuse Release event with Push event.</p>
<p>You also can follow my suggestions mentioned in that ticket to solve the problem:</p>
<ol>
<li>
<p>If you want to trigger the release workflow on the <strong>patch-1</strong> branch, when you create a new release, you should select the tag created from the <strong>patch-1</strong> branch.</p>
</li>
<li>
<p>If you create release from the <strong>master</strong> branch, and want to trigger the release workflow, you should make sure the YAML file also has been existing on the <strong>master</strong> branch, and select the tag created from the <strong>master</strong> branch when creating a new release.</p>
</li>
</ol>
